go/tasks/pluginmachinery/flytek8s/pod_helper.go

```diff
@@ -212,6 +215,9 @@ func ConvertPodFailureToError(status v1.PodStatus) (code, message string) {
 					c.LastTerminationState.Terminated.ExitCode,
 					c.LastTerminationState.Terminated.Reason,
 					c.LastTerminationState.Terminated.Message)
+			} else if c.State.Terminated != nil && strings.Contains(c.State.Terminated.Reason, OOMKilled) {
```

Comment:
Is this correct? as in `c.LastTerminationState.Terminated` will be nil in the case of OOMKilled? have you verified that?

Reply:
yep, see: https://gist.github.com/katrogan/a4711f31af0af4b851195e5012313af0 for an example of the full pod output